docs/application/native/guides/app-management/app-controls.md

```diff
@@ -107,6 +107,10 @@ else
 
 app_control_destroy(app_control);
 ```
+> **Note**
+>
+> From Tizen 5.0 and onwards, two APIs whose callbacks have a result were added to return the result.
```

Comment:
@nawab-samsung 
I have a suggestion to this update.

1. How about using "Since Tizen 5.0" instead of "From Tizen 5.0 and onwards," regarding [here](https://github.com/Samsung/tizen-docs/pull/556#discussion_r258824227)?

2. How about listing the items?

> **Note**
>
> Since Tizen 5.0, two APIs whose callbacks have a result were added to return the result.
>
> - `app_control_send_launch_request_async()` : Asynchronously launches application controls.
> - `app_control_send_launch_request_sync()` : Synchronously launches application controls.

Reply:
The app_control_send_launch_request_sync() is supported since tizen 5.5. 

Hmm...

Reply:
I've updated the description.
Please check and review it.

Reply:
Sure @hjhun  is this final change from your side ?